atomic_reactor/plugins/exit_delete_from_registry.py

```diff
@@ -37,7 +39,22 @@ def __init__(self, tasker, workflow, registries):
         """
         super(DeleteFromRegistryPlugin, self).__init__(tasker, workflow)
 
-        self.registries = deepcopy(registries)
+        if check_variable(self.workflow, get_registries):
+            registries_cm = {}
+            all_registries = get_registries(self.workflow)
+            for registry in all_registries:
+                reguri = RegistryURI(registry.get('url'))
+                regdict = {}
+                regdict['version'] = reguri.version
+                if registry.get('auth'):
+                    regdict['secret'] = registry['auth']['cfg_path']
+                regdict['insecure'] = registry.get('insecure', False)
+
+                registries_cm[reguri.docker_uri] = regdict
+
+            self.registries = deepcopy(registries_cm)
+        else:
+            self.registries = deepcopy(registries)
```

Comment:
This code is repeated in several other plugins. It should be refactored and moved into util.py